lib/sync-settings.coffee

```diff
@@ -53,7 +53,9 @@ module.exports =
       cb?(err, res)
 
   getPackages: ->
-    for name,info of atom.packages.getLoadedPackages()
+    packages = atom.packages.getLoadedPackages()
+    for name,info of packages
+      continue unless packages.hasOwnProperty(name)
```

Comment:
Isn't this querying whether the packages array has a name property rather than the current iteration?